meeting-notes/2019-12-16--libp2p-sync.md

```diff
@@ -0,0 +1,71 @@
+# 2019-12-16 libp2p Sync Call
+
+[Issue for this call](https://github.com/libp2p/team-mgmt/issues/16)
+
+**Moderator**: @jacobheun
+
+**Notetaker**: @jacobheun
+
+**Recording**: https://youtu.be/GNR3hGs2UKY
+
+**Attendees**:
+  - @vasco-santos
+  - @aschmahmann
+  - @vyzo
+
+
+## Agenda
+
+- Weekly updates 
+
+## Weekly updates
+
+- @me:
+  - Done:
+  - Blocked:
+  - Next:
+
+
+- @jacobheun
+  - Done:
+    - libp2p async refactor
+      - Released a pre release
+      - Added coalescing dial support [js-libp2p#518](https://github.com/libp2p/js-libp2p/pull/518)
+      - Integrated pre release into js-ipfs-bitswap [js-ipfs-bitswap#206](https://github.com/ipfs/js-ipfs-bitswap/pull/206)
+      - Misc fixes and documentation updates as part of downstream migration
+  - Next:
+    - Integrate libp2p pre release into js-ipfs
+    - Finish reviewing all the libp2p examples
+    - Create a migration guide from libp2p@0.26.x to libp2p@0.27.x
+    - Do a final pass through the API and Configuration docs
+    - Get an RC of libp2p out.
+
+- @vasco-santos
+  - Done:
+    - API and CONFIG docs merged
+    - getting started first iteration
+      - [libp2p/js-libp2p#514](https://github.com/libp2p/js-libp2p/pull/514)
+    - async refactor for ping
+      - [libp2p/js-libp2p#505](https://github.com/libp2p/js-libp2p/pull/505)
+    - js-libp2p release fixes
+      - [libp2p/js-libp2p#510](https://github.com/libp2p/js-libp2p/pull/510)
+      - [libp2p/js-libp2p#513](https://github.com/libp2p/js-libp2p/pull/513)
+      - [libp2p/js-libp2p#517](https://github.com/libp2p/js-libp2p/pull/517)
+    - more examples refactored
+      - [libp2p/js-libp2p#507](https://github.com/libp2p/js-libp2p/pull/507)
+      - [libp2p/js-libp2p#508](https://github.com/libp2p/js-libp2p/pull/508)
+    - awesome-libp2p proposal 
+  - Blocked: N/A
+  - Next:
+    - New iteration on some `js-libp2p` examples
+    - Finish getting started guide and blogpost
+    
+- @aschmahmann
+   - Blocked:
+      - Provider records using multihashes instead of CIDs (https://github.com/libp2p/go-libp2p-kad-dht/pull/422)
+        - Blocked from being merged awaiting consensus on what types of provider record keys DHT servers should allow (multihashes, multihashes + CIDs, anything, anything less than 80 bytes, etc.)
+      - @raulk asked me to review [kad#383](https://github.com/libp2p/go-libp2p-kad-dht/pull/383). There are a few changes since he last saw it. Can we (@aarsh + I) just use our best judgment merge or should we wait for @raulk to review?
```

Comment:
@aschmahmann thanks for pushing this! I’ll review this week for final approval :-)